samples/updates.html

```diff
@@ -151,19 +153,19 @@
 		};
 
 		function addLine() {
-			window.myScatter.options.plugins.annotation.annotations.push({
+			window.myScatter.options.plugins.annotation.annotations.myHorizontlLine = {
```

Comment:
This does not work as it used to

Reply:
I have changed the logic, my fault.

I'm gonna change building a id based on a counter.